modules/rest/openapi/openapi.yaml

```diff
@@ -319,6 +319,209 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /management/v1/deployment/units:
+    get:
+      tags:
+      - deployment
+      description: All units statutes.
+      operationId: units
+      parameters: []
+      responses:
+        "200":
+          description: All statutes returned successful.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UnitStatus'
+        "500":
+          description: Internal error.
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+    post:
+      tags:
+      - deployment
+      description: Deploy provided unit to node.
+      operationId: deployUnit
+      parameters: []
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              required:
+              - unitContent
+              - unitId
+              type: object
+              properties:
+                unitId:
+                  required:
+                  - "true"
+                  type: string
+                unitVersion:
+                  type: string
+                unitContent:
+                  required:
+                  - "true"
+                  type: string
+                  format: binary
+        required: true
+      responses:
+        "200":
+          description: Unit deployed successfully.
+          content:
+            application/json:
+              schema:
+                type: boolean
+        "500":
```

Comment:
What if there is already a unit with the same is and version? I think we return 409 but its not mentioned in the spec.